Software/Development/SemesterProject/System/Include/scheduler.h

```diff
@@ -35,8 +36,12 @@ typedef struct TCBstruct
 void taskListInitialization(void);
 void scheduleTasks(void);
 void runTask(void (*functionPTR)());
+
 void task50us(void);
 void task20ms(void);
+void task200ms(void);
```

Comment:
Is this the minimum freq at which the GUI can get  updated?

Reply:
yes-ish. Maybe further research shows that it can be faster.